cmd/csaf_validator/main.go

```diff
@@ -26,6 +26,7 @@ type options struct {
 	RemoteValidator        string   `long:"validator" description:"URL to validate documents remotely" value-name:"URL"`
 	RemoteValidatorCache   string   `long:"validatorcache" description:"FILE to cache remote validations" value-name:"FILE"`
 	RemoteValidatorPresets []string `long:"validatorpreset" description:"One or more presets to validate remotely" default:"mandatory"`
+	Verbose                bool     `long:"verbose" description:"If a remote validator was used, display the results in JSON format"`
```

Comment:
In the other programs `-v` is additionally short for  `--verbose`

Reply:
Along with renaming verbose to output due to the concerns in https://github.com/csaf-poc/csaf_distribution/pull/347#issuecomment-1452470345, the -o flag has now been added